RELEASE/scripts/autoscend/iotms/mr2022.ash

```diff
@@ -137,3 +137,84 @@ boolean auto_fightLocketMonster(monster mon)
 	return true;
 
 }
+
+void juneCleaverChoiceHandler(int choice)
+{
+	switch(choice) {
+		case 1467: // Poetic Justice
+			run_choice(3); // +5 adventures, get beaten up
+			break;
+		case 1468: // Aunts not Ants
+			if(my_primestat() == $stat[muscle] && my_level() < 13) {
+				run_choice(2); // 250 muscle substat
+			} else if(my_primestat() == $stat[moxie] && my_level() < 13) {
+				run_choice(1); // 150 moxie substat
+			// } else if(check a property to see if skips are available) {
+			// 	run_choice(4); // skip
+			} else {
+				run_choice(2); // 250 muscle substat
+			}
+			break;
+		case 1469: // Beware of Alligators
+			if (can_drink() && my_inebriety() < inebriety_limit()) {
```

Comment:
Thoughts on adding a check above this for current meat < meat reserve, if it is pick 1500 meat? I think of we are struggling to have enough meat, that would be better than an awesome booze